webpack.common.js

```diff
@@ -92,8 +97,10 @@ module.exports = {
             chunks: 'all',
             maxInitialRequests: Infinity,
             cacheGroups: {
-                vendor: {
-                    test: /[\\/]node_modules[\\/]/,
+                node_modules: {
+                    test(module) {
+                        return NODE_MODULES_REGEX.test(module.context);
```

Comment:
The previous test caused a crash when using the mini css plugin. This test should be basically the same, but doesn't crash.